torch/onnx/symbolic_helper.py

```diff
@@ -970,6 +970,12 @@ def requantize_bias_helper(g, bias, input_scale, weight_scale):
                   to_i=torch.onnx.TensorProtoDataType.INT32)
     return g.op("prim::TupleConstruct", q_bias, bias_scale, bias_zero_point)
 
+def check_args_have_same_dtype(args):
```

Comment:
Equivalent and shorter name, also makes it obvious what the return type is from the name:
```suggestion
def args_have_same_dtype(args):
```